docs/content/docs/dev/table/materialized-table/syntax.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Syntax
+weight: 2
+type: docs
+aliases:
+- /dev/table/materialized-table/syntax.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Description
+
+This document explains the syntax and usage of materialized tables.
```

Comment:
Flink SQL supports the following Materialized Table statements for now:

- CREATE MATERIALIZED TABLE
- ALTER MATERIALIZED TABLE
- DROP MATERIALIZED TABLE